### PR TITLE
feat: supporting transactions

### PR DESCRIPTION
**Pull request by jleon15**
## Description

- Support transactions
 

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

- [x] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: src/types/sdk/services/transactions.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import type { RequestOptions } from '@/types/sdk';
+
+type CreateTransactionResponse = {
+  id: string;
+  createdBy: string;
+  createdDate: string;
```
Comment (dhudec) [inline src/types/sdk/services/transactions.ts:6]: This is returned from the api as `created_at`, which would need to translate into `createdAt` with the camel case conversion here.

File: test/transactions.test.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        [API_KEY_HEADER]: expect.any(String),
+      });
+    });
+  });
+
+  describe('commit', () => {
+    test('should commit transaction', async () => {
+      const id = chance.string();
+
+      client.onPost(`/${id}/commit`).reply(200, JSON.stringify({}));
```
Comment (dhudec) [inline test/transactions.test.ts:62]: We were going to change these status codes to 204's, right?

File: src/types/sdk/services/transactions.ts (hunk 1 above)
Comment (jleon15) [reply src/types/sdk/services/transactions.ts]: nice catch

**Review by dhudec — APPROVED** [review]

**Pull request merged**